src/nav/apm.yml

```diff
@@ -1103,79 +1103,105 @@ pages:
       - title: Ruby monitoring
         path: /docs/apm/agents/ruby-agent
         pages:
-          - title: Monitor your Ruby app
+          - title: Introduction to Ruby monitoring
             path: /docs/apm/agents/ruby-agent/getting-started/introduction-new-relic-ruby
+          - title: Install the Ruby agent
+            path: /docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent
+          - title: Ruby configuration
+            path: /docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration
+          - title: Ruby custom instrumentation
```

Comment:
Not sure about pulling this one up so prominently, is this traffic-driven or...?

Reply:
Which one, the custom instrumentation or the configuration? I moved the custom instrumentation up because it's one of the most highly viewed docs and, according to the Ruby team, well used.

Reply:
The "custom instrumentation" one. It's definitely high-traffic, but to me the top layer of the nav is for newbie users primarily. I worry it will create the impression this is a required process, or otherwise add noise for brand-new users

Reply:
I understand where you're coming from, @austin-schaefer and would be comfortable with keeping "Ruby custom instrumentation" a level down.